examples/node-app-mssql/README.md

```diff
@@ -10,12 +10,75 @@ Install dependencies.
 $ yarn
 ```
 
-Configure database connection(s) in the `connections.sync-db.json`. 
+Configure database connection(s) in the `connections.sync-db.json`.
 
 ```bash
 $ cp connections.sync-db.json.example connections.sync-db.json
 ```
 
+## Migrations
+
+Create a configuration file `knexfile.ts` in the root directory.
+
+```js
+import { connections } from './connections.sync-db';
+
+module.exports = {
+  client: 'mssql',
+  connection: connections[0],
+  migrations: {
+    directory: './src/sql/migrations',
```

Comment:
Please add the migration files as well. Where is the code? Do we not need it?

cc @kabirbaidhya

Reply:
To add migration files the [Migration](https://github.com/p0k8h/sync-db/tree/sample-tables-using-knex-migrations/examples/node-app-mssql#migrations) procedure should be followed.

Which will create a new `knexfile.ts` config file and adding scripts on `package.json`.

Should I proceed?